src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -26,6 +28,15 @@ def get(self, request, organization):
             organization=organization,
         ).select_related('user').order_by('email', 'user__email')
 
+        query = request.GET.get('query')
+        if query:
+            tokens = tokenize_query(query)
+            for key, value in six.iteritems(tokens):
+                if key == 'email':
+                    queryset = queryset.filter(
+                        user_id__in=User.objects.filter(email__in=value).values('id')
```

Comment:
So you can use `user__email` here instead of `user_id`. You'll also want to check through the `UserEmail`s as well (since a user can have multiple emails), so you can do something like this:

```
queryset = queryset.filter(
    Q(user__email__in=value) | Q(user__emails__email__in=value)
)
```